storagenode/peer.go

```diff
@@ -149,6 +149,13 @@ func (peer *Peer) Run(ctx context.Context) error {
 	defer cancel()
 
 	var group errgroup.Group
+	group.Go(func() error {
+		err := peer.Public.Server.Run(ctx)
```

Comment:
@egonelbre This is the only thing I think I found that might be of value... Kademlia has a 'Ping back' feature, so Kademlia's gRPC should always be started up before Kademlia bootstraps.

Reply:
Well, since they are started in parallel the slight ordering change between the goroutines shouldn't change much.